test/e2e/e2e_test.go

```diff
@@ -232,7 +232,9 @@ func testDisasterRecoveryWithStorageType(t *testing.T, numToKill int, bt spec.Ba
 		if err := deleteEtcdCluster(f, testEtcd.Name); err != nil {
 			t.Fatal(err)
 		}
-		// TODO: add checking of removal of backup pod
+		if err := checkPVBackupDeleted(f, testEtcd.Name); err != nil {
```

Comment:
is this correct? the func called testDisasterRecoveryWithStorageType. why we have a pv specific func here? it seems we should do this outside this func in `TestDisasterRecoveryWithPV`